### PR TITLE
Added calander layout

### PR DESCRIPTION
**Pull request by Vishalk25**
image

### CODE REVIEW

File: src/components/Layouts/Calander/Calander.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Calander.css
+
+.calander-main {
+  margin-left: "18%";
+  height: "60%x";
```
Comment (SaurabhK05) [inline src/components/Layouts/Calander/Calander.css]: Remove x from height

**Review by SaurabhK05 — DISMISSED** [review]

**Comment by SaurabhK05** [comment]
@Vishalk25 , Please take merge from master and use new components. Kindly use 'SideBar', 'Header' component to create Calender page

**Comment by SaurabhK05** [comment]
For File naming convention, we should follow Figma page names in the layout pages, 
For e.g:
**Layouts/EventsPage/Event.js**
This must be your layout page, instead of this
image
And same for .CSS file

File: src/components/Layouts/Calander/Eventcard.js
Comment (SaurabhK05) [inline src/components/Layouts/Calander/Eventcard.js:1]: Please move this file to Common components folder

**Review by TeamMonuKumar — APPROVED** [review]
need to change svg image path

**Pull request merged**